.cirrus.yml

```diff
@@ -26,6 +26,10 @@ task:
       script:
         - flutter channel master
         - ./script/check_publish.sh
+    - name: version
+      script:
+        - flutter channel master
+        - ./script/check_version.sh
```

Comment:
Let's fold this into the `publishable` task a new script line. There's some overhead to each job we spin up, and this is conceptually part of `publishable` IMO.